src/Google.Datastore.V1Beta3/QueryExtensions.cs

```diff
@@ -64,5 +64,16 @@ public static void Add(this RepeatedField<PropertyOrder> orderings, string prope
                 nameof(direction), "Direction must be Ascending or Descending");
```

Comment:
I know it's not relevant to this change, but why doesn't this CheckArgument use the enum checker?